client/my-sites/posts/main.jsx

```diff
@@ -73,13 +74,18 @@ const PostsMain = React.createClass( {
 					siteId={ site.ID }
 					query={ this.props.draftsQuery } />
 				<QueryPostCounts siteId={ site.ID } type="post" />
-				<Card compact className="posts__drafts-header">
-					{ this.translate( 'Latest Drafts' ) }
-					<Button borderless href={ this.props.newPostPath }>
-						<Gridicon icon="plus" />
+				<SectionHeader className="posts__drafts-header" label={ this.translate( 'Latest Drafts' ) }>
+					<Button compact href={ this.props.newPostPath }>
+						{ this.translate( 'Start New' ) }
 					</Button>
-				</Card>
-				{ this.props.drafts && this.props.drafts.map( this.renderDraft, this ) }
+				</SectionHeader>
+				{ map( this.props.drafts, ( draft ) => (
+					<PostItem
+						compact
+						key={ draft.global_ID }
+						globalId={ draft.global_ID }
+					/>
+				) ) }
 				{ isLoading && <Draft isPlaceholder /> }
```

Comment:
Why not `<PostItem />` so we can drop the `<Draft />` dependency?

From docs:

>If [`globalId` is] omitted, it's assumed that the post is currently loading, and a post with placeholder styling will be shown instead.

https://github.com/Automattic/wp-calypso/tree/master/client/blocks/post-item